index.html

```diff
@@ -89,5 +89,225 @@ <h1 class="headline-title">Hey There.<br />I’m Htet.</h1>
+          like what you see and have a project you need coded, don’t hesitate to
+          contact me.
+        </p>
+        <a class="get-my-resume-btn" href="#">Get My Resume</a>
+      </div>
+    </section>
+    <section class="skills-section">
+      <div class="about-me-line"></div>
+
+      <ul class="skills-wrapper">
+        <li>
+          <ul class="skill-wrapper">
+            <li class="skill-title-wrapper">
+              <svg
+                class="skill-title-svg"
+                width="27"
+                height="26"
+                viewBox="0 0 27 26"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <rect
+                  x="0.772461"
+                  y="12.7279"
+                  width="18"
+                  height="18"
+                  transform="rotate(-45 0.772461 12.7279)"
+                  fill="#FF6B00"
+                />
+              </svg>
+              <h3 class="skill-title">Language</h3>
+            </li>
+            <li class="skill">JavaScript</li>
+            <li class="skill">Ruby</li>
+            <li class="skill">HTML</li>
+            <li class="skill">CSS</li>
+          </ul>
+        </li>
+        <li>
+          <ul class="skill-wrapper">
+            <li class="skill-title-wrapper">
+              <svg
+                class="skill-title-svg"
+                width="18"
+                height="19"
+                viewBox="0 0 18 19"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <rect y="0.455811" width="18" height="18" fill="#FF6B00" />
+              </svg>
+
+              <h3 class="skill-title">Frameworks</h3>
+            </li>
+            <li class="skill">React.js</li>
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">RSpec</li>
+            <li class="skill">Capybara</li>
+          </ul>
+        </li>
+        <li>
+          <ul class="skill-wrapper">
+            <li class="skill-title-wrapper">
+              <svg
+                class="skill-title-svg"
+                width="18"
+                height="19"
+                viewBox="0 0 18 19"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <rect
+                  y="0.455811"
+                  width="18"
+                  height="18"
+                  rx="9"
+                  fill="#FF6B00"
+                />
+              </svg>
+
+              <h3 class="skill-title">Skills</h3>
+            </li>
+
+            <li class="skill">Database Management</li>
+            <li class="skill">Version Control</li>
+            <li class="skill">CLI</li>
+            <li class="skill">API Design</li>
+            <li class="skill">Web Development</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
```

Comment:
- [ ] Just as a friendly reminder, it would be best to consolidate these sections into one cohesive "About Me" section. By doing this, the information provided will be easier to navigate and more organized for the reader. Additionally, having all the relevant details about yourself in one place will allow the reader to get a more comprehensive understanding of who you are and what you have to offer. This will ultimately create a more positive user experience and increase the chances of the reader engaging with your portfolio :+1:.